### PR TITLE
Fast documentation updates,pr2 completed

### PR DESCRIPTION
**Pull request by xwaeaewcrhomesysplug**
for pytorch engine in slack channel help.does not work fix
Overall documentaion

A very fast documentation pr not much info

### CODE REVIEW

File: docs/JavatoNative_EngineMapping.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<JavaEngine>:<Native Engine>
```
Comment (stu1130) [inline docs/JavatoNative_EngineMapping.txt:1]: what is this doc for

File: pytorch/pytorch-engine/README.md [hunk 2]
```diff
@@ -1,195 +1,195 @@
...
+We offer an automatic option that will download the native libraries into [cache folder](../../docs/development/cache_management.md) the first time you run DJL.
+It will automatically determine the appropriate jars for your system based on the platform and GPU support.
+
+- ai.djl.pytorch:pytorch-native-auto:1.6.0
+
+```xml
+<dependency>
+    <groupId>ai.djl.pytorch</groupId>
+    <artifactId>pytorch-native-auto</artifactId>
+    <version>1.6.0</version>
```
Comment (stu1130) [inline pytorch/pytorch-engine/README.md:53]: we are going to release 0.9.0 which works with pytorch 1.7.0 after Thanksgiving. Meanwhile, you can leverage 0.9.0-SNAPSHOT to try pytorch 1.7.0

**Comment by xwaeaewcrhomesysplug** [comment]
Eh,as stated it is a documentation of solution in slack hekp channel.If there are possiblities of forward compatible binaries.Feel free to include it into the documentations. I think it will generally gives a more referable and documented solution in long run. For example ,now is v0.80. I do not think you remembered all the mapping in the v060 or etc. Automation could also be made utilizing this documentation with a simple parser.You may kick this pr if others also think it is bloated or actually have existing solution. Sorry,quite new here so many have not read some of the existing documentation.

File: pytorch/pytorch-engine/README.md (hunk 2 above)
Comment (xwaeaewcrhomesysplug) [reply pytorch/pytorch-engine/README.md:53]: This yeah can actually not update as long as user knows they need to use snapshot or the lower version.Just a side edit.Can be scraped.

**Comment by xwaeaewcrhomesysplug** [comment]
stale pr take down.

**Pull request closed without merging**